src/ServiceControl.PersistenceTests/PersistenceTestCollection.cs

```diff
@@ -0,0 +1,15 @@
+namespace ServiceControl.Persistence.Tests
+{
+    using System.Collections;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    public class PersistenceTestCollection : IEnumerable
+    {
+        public IEnumerator GetEnumerator()
+        {
+            yield return new PersistenceDataStoreFixture(DataStoreType.InMemory);
+            yield return new PersistenceDataStoreFixture(DataStoreType.RavenDb);
+            yield return new PersistenceDataStoreFixture(DataStoreType.SqlDb);
+        }
+    }
+}
```

Comment:
This is where we need that check for the existence of the sqlconnection in env variables

Reply:
Updated